roofit/roofitcore/test/TestStatistics/testRooAbsL.cxx

```diff
@@ -293,7 +293,7 @@ TEST_F(SimBinnedConstrainedTest, SubEventSections)
          {static_cast<double>(ix) / N_events_total, static_cast<double>(ix + 1) / N_events_total}, 0,
          likelihood->getNComponents());
    }
-   EXPECT_EQ(whole.Sum(), N_events_total_parts.Sum());
+   EXPECT_DOUBLE_EQ(whole.Sum(), N_events_total_parts.Sum());
```

Comment:
```suggestion
   // We cannot EXPECT_EQ in this test, because we compare actually different
   // calculations. The multiple additions and FMA operations involved in the
   // calculation of the multiple parts introduces different rounding errors
   // on the CPU level than the single calculation over all events at once.
   EXPECT_DOUBLE_EQ(whole.Sum(), N_events_total_parts.Sum());
```